dll/win32/t2embed/t2embed.spec

```diff
@@ -11,15 +11,15 @@
 @ stdcall TTRunValidationTests(ptr ptr)
 @ stub _TTCharToUnicode@24
 @ stub _TTDeleteEmbeddedFont@12
-@ stdcall _TTEmbedFont@44(long long long ptr ptr ptr ptr ptr long long ptr) TTEmbedFont
+@ stdcall -i386 _TTEmbedFont@44(long long long ptr ptr ptr ptr ptr long long ptr) TTEmbedFont
 @ stub _TTEmbedFontFromFileA@52
 @ stub _TTEnableEmbeddingForFacename@8
 @ stub _TTGetEmbeddedFontInfo@28
-@ stdcall _TTGetEmbeddingType@8(long ptr) TTGetEmbeddingType
-@ stdcall _TTIsEmbeddingEnabled@8(long ptr) TTIsEmbeddingEnabled
-@ stdcall _TTIsEmbeddingEnabledForFacename@8(str ptr) TTIsEmbeddingEnabledForFacename
-@ stdcall _TTLoadEmbeddedFont@40(ptr long ptr long ptr ptr ptr wstr str ptr) TTLoadEmbeddedFont
-@ stub _TTRunValidationTests@8
+@ stdcall -i386 _TTGetEmbeddingType@8(long ptr) TTGetEmbeddingType
+@ stdcall -i386 _TTIsEmbeddingEnabled@8(long ptr) TTIsEmbeddingEnabled
+@ stdcall -i386 _TTIsEmbeddingEnabledForFacename@8(str ptr) TTIsEmbeddingEnabledForFacename
+@ stdcall -i386 _TTLoadEmbeddedFont@40(ptr long ptr long ptr ptr ptr wstr str ptr) TTLoadEmbeddedFont
+@ stub -i386 _TTRunValidationTests@8
 @ stdcall TTEmbedFontEx(ptr long long ptr ptr ptr ptr ptr long long ptr)
```

Comment:
Please allow me to learn something:
"Remove stdcall annotated exports from x64" <- you obviously did not chose to add the -i386 after all that are stdcall. I am pretty sure, that you know what you are doing, so allow me to understand: What is the additional condition that must be met, that it has to be placed here?
-386 will not export them for x64. So is the additional condition, the 'secret' knowledge that I do lack, just that those exports do not exist on x64? Or is there more about it? Thx!

Reply:
Looking at it, my guess would be that the excluded ones are forwarders.
If so, another question could be: why exclude `_TTRunValidationTests@8` but not the other decorated stubs?
I am not saying anything is wrong either, only confirming that PR purpose is a bit unclear.

Reply:
The keyword was "annotated" (exports), not "stdcall". The `_Whatever@8(...)` type of decoration is x86 32-bits specific.

Reply:
> `_Whatever@8(...)` type of decoration is x86 32-bits specific.

Thanks for the more explicit explanation.
Then, `_TTEnableEmbeddingForFacename@8`, at least, should be excluded too, shouldn't it?